src/hooks/useBorrowRepayModal/Modal/index.tsx

```diff
@@ -5,19 +5,41 @@ import { useTranslation } from 'translation';
 import { Asset } from 'types';
 import { isAssetEnabled } from 'utilities';
 
-import { useStyles } from '../styles';
+import { useGetUserMarketInfo } from 'clients/api';
+import { AuthContext } from 'context/AuthContext';
+
 import Borrow from './Borrow';
 import Repay from './Repay';
+import { useStyles } from './styles';
 
 export interface BorrowRepayProps {
   onClose: ModalProps['handleClose'];
   isXvsEnabled: boolean;
-  asset: Asset;
+  assetId: Asset['id'];
 }
 
-const BorrowRepay: React.FC<BorrowRepayProps> = ({ onClose, asset, isXvsEnabled }) => {
+const BorrowRepay: React.FC<BorrowRepayProps> = ({ onClose, assetId, isXvsEnabled }) => {
   const { t } = useTranslation();
   const styles = useStyles();
+  const { account } = React.useContext(AuthContext);
+
+  // TODO: handle loading state (see https://app.clickup.com/t/2d4rcee)
```

Comment:
We aren't using clickup anymore so this can be removed or updated with a JIRA ticket

Reply:
Good catch